user/cron-jobs.md

```diff
@@ -27,6 +25,10 @@ Confirm that the cron job is displayed in your settings tab:
 
 ![cron job created](/images/cron-created.png "cron job created")
 
+## Skipping Cron Jobs
+
+Because cron jobs build the latest commit to their branch, if a commit message includes [`[ci skip]` or `[skip ci]`](/user/customizing-the-build/#Skipping-a-build) it will not be built. 
```

Comment:
I think we need to add a little more info here. What we're actually saying is "when a cron job would otherwise run, if the latest commit includes `skip` then it will not run."

Maybe just saying 

> Because cron jobs build the latest commit to a particular branch, if that commit message includes [`[ci skip]` or `[skip ci]`](/user/customizing-the-build/#Skipping-a-build) the cron job will skip that build.

is enough? @acnagy what do you think?